_maps/map_files/MiniStation/MiniStation.dmm

```diff
@@ -12882,6 +12884,14 @@
 	},
 /turf/simulated/floor/plasteel,
 /area/engine/engineering)
+"AR" = (

```

Comment:
This specifically will conflict with your other PR, but not much can be done about that.